src/components/Radio/Radio.tsx

```diff
@@ -48,11 +48,7 @@ class Radio extends UIComponent<any, any> {
     return (
       <ElementType {...rest} className={classes.root}>
         <Label styles={{ root: styles.label }}>
```

Comment:
at the same time, if we are agreeing to eliminate `Input` composition - then I do have a question why we would need `Label` component here? to me it seems a bit counterintuitive, as `Label` semantically doesn't correspond to `form`'s label